### DISCUSSION TITLE
Need to group my rows for 5 to 6 columns rowspan

### DISCUSSION THREAD

**Discussion by Mertkmrc** [primefaces/community#926] [PrimeNG]
As the title says, I need to group my rows with rowspan. However only one column will have different values. I found the 'groupRowsBy" parameter but it groups for only one column. Is there any easy way to accomplish this?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please check the row-group examples https://primeng.org/table#row-group? It's not clear what you're trying to achieve, maybe sharing a stackblitz example would be helpful.
